experimental/handoff-library/csharp_dotnetcore/samples/LivePersonAgentBot/PostDeployScripts/mergeSettings.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs');
+const path = require('path');
+const appSettingPath = path.resolve('./appsettings.json');
+const envPath = path.resolve('./.env');
```

Comment:
This file is quite complex. None of our existing samples do anything like this. Is this something we need here?